src/libcore/marker.rs

```diff
@@ -351,7 +351,35 @@ pub trait PhantomFn<A:?Sized,R:?Sized=()> { }
 /// instance, it will behave *as if* an instance of the type `T` were
 /// present for the purpose of various automatic analyses.
 ///
-/// For example, embedding a `PhantomData<T>` will inform the compiler
+/// # Example
```

Comment:
We use 'Examples' for everything, even if there's just one. And, there needs to be a newline after it as well.